hs_tools_resource/page_processors.py

```diff
@@ -50,7 +50,7 @@ def landing_page(request, page):
             context['supported_sharing_status'] = sharing_status_str
 
         if content_model.metadata.tool_icon.first():
-            context['tool_icon_url'] = content_model.metadata.tool_icon.first().value
+            context['tool_icon_url'] = content_model.metadata.tool_icon.first().data_url
```

Comment:
@zhiyuli It appears both value and data_url is used and it appears value can contain a url as well. Can value and data_url be combined into a field? Most likely I don't understand the use case here, but from what I can see, it appears one field could be sufficient.